src/hazem-components/PageN5-components/Content.jsx

```diff
@@ -1,30 +1,24 @@
-
+import Data from '../../MainData.json';
```

Comment:
Make the component itself take props and add the data from the Page 
so if we have as following:
Pages/
 |-Home.jsx
 |-about.jsx
 Component
|-CompOne.jsx
|-CompTWO.jsx
|-Comp3.jxs
Data
|-JSONDATA.json
### If we said that the home page will take 2 components "CompOne, CompTWO" then import the "JSONDATA.json" at the Home Page and Pass the data to the 2 components and make each component takes Props

> so you need to import the Data at Home Page